lib/web_ui/test/engine/platform_dispatcher_test.dart

```diff
@@ -18,6 +18,21 @@ void main() {
 
 void testMain() {
   group('PlatformDispatcher', () {
+    test('high contrast in accessibilityFeatures has the correct value', () {
+      final MockHighContrastSupport mockHighContrast =
+          MockHighContrastSupport();
+      HighContrastSupport.instance = mockHighContrast;
+      final EnginePlatformDispatcher engineDispatcher =
+          EnginePlatformDispatcher();
+
+      expect(engineDispatcher.accessibilityFeatures.highContrast, true);
```

Comment:
It's better to use `isTrue` in tests:
```suggestion
      expect(engineDispatcher.accessibilityFeatures.highContrast, isTrue);
```